config/default.yaml

```diff
@@ -10,9 +10,10 @@ config:
     #    credentials: username:password
         baseUrl: https://jenkins.example.com
         saveCache: true
-        projectFormat: build-{project:alpha}-snapshot
+        projectFormat: '{project:alpha}'
         # promotion environments
         environments:
+            prd_aws: prd
```

Comment:
why not `prd` instead of `prd_aws`?

Reply:
this maps prd_aws environment for old prd promotions on an old jenkins onto the prd tag

Reply:
;)